core/tauri-utils/src/config.rs

```diff
@@ -1872,6 +1872,94 @@ impl<'de> Deserialize<'de> for UpdaterEndpoint {
   }
 }
 
+/// Install modes for the Windows update.
+#[derive(Debug, PartialEq, Clone)]
+#[cfg_attr(feature = "schema", derive(JsonSchema))]
+#[cfg_attr(feature = "schema", schemars(rename_all = "camelCase"))]
+pub enum WindowsUpdateInstallMode {
+  /// Specifies there's a basic UI during the installation process, including a final dialog box at the end.
+  BasicUi,
+  /// Specifies there's no UI during the installation process.
+  NoUi,
+  /// The quiet mode means there's no user interaction required.
+  /// Requires admin privileges if the installer does.
+  Quiet,
+  /// Specifies unattended mode, which means the installation only shows a progress bar.
+  Passive,
+}
+
+impl WindowsUpdateInstallMode {
+  /// Returns the associated `msiexec.exe` arguments.
+  pub fn msiexec_args(&self) -> &'static [&'static str] {
+    match self {
+      Self::BasicUi => &["/qb+"],
+      Self::NoUi => &["/qn"],
+      Self::Quiet => &["/quiet"],
```

Comment:
I'm like 95% sure that /qn and /quiet is the same. i think /quiet is just "newer".

Reply:
you're right, gonna remove it